Plugins/Public/mobiledocking_plugin/Main.cpp

```diff
@@ -614,64 +681,30 @@ bool UserCmd_Process(uint client, const wstring &wscCmd)
 	return false;
 }
 
-void __stdcall JumpInComplete(uint iSystemID, uint iShip)
-{
-	returncode = DEFAULT_RETURNCODE;
-	const uint iClientID = HkGetClientIDByShip(iShip);
-	
-	// After completing a jump, if it's in our dockedships map, we update the current system
-	if(mobiledockClients.find(iClientID) != mobiledockClients.end())
-	{
-		pub::SpaceObj::GetSystem(iShip, mobiledockClients[iClientID].carrierSystem);
-	}
-	
-}
-
-void __stdcall CharacterSelect(struct CHARACTER_ID const & cId, unsigned int iClientID)
-{
-
-	returncode = DEFAULT_RETURNCODE;
-	
-	// Attempt to load the data for this ship if it exists. Trim off the .fl at the end of the file name.
-	string charFilename = cId.szCharFilename;
-	charFilename = charFilename.substr(0, charFilename.size() - 3);
-
-	LoadShip(charFilename);
-
-}
-
 void __stdcall DisConnect(uint iClientID, enum EFLConnection p2)

```

Comment:
While you can use DisConnect for this, the convention is to handle the cleaning of client data on the connection of a new client. That's why ClearClientInfo exists.

Reply:
Makes sense. Removed.